kairon/shared/pyscript/callback_pyscript_utils.py

```
@@ -335,4 +335,76 @@ def save_as_pdf(text: str, bot: str, sender_id:str):
             )
             return media_id
         except Exception as e:
-            raise Exception(f"encryption failed-{str(e)}")
\ No newline at end of file
+            raise Exception(f"encryption failed-{str(e)}")
+
+    @staticmethod
+    def add_analyitcs_raw_data(collection_name: str, bot: Text = None):
```

Ruff (0.14.5)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)